core-java/src/test/java/com/baeldung/nulls/PrimitivesAndWrapperUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.nulls;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class PrimitivesAndWrapperUnitTest {
```

Comment:
When using the given-when-then pattern, it's givenCondition_whenDoSomething_thenExpectedBehavior. Several of the unit tests here and in other classes have the given and when reversed. For example, the first one could be called something like givenBothArgsNonNull_whenCallingWrapperSum_thenReturnSum.